packages/salesforcedx-vscode-soql/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# 50.14.0 - January 21, 2021
+
+## Added
+
+- SOQL code-completion on WHERE clause expressions ([PR #2839](https://github.com/forcedotcom/salesforcedx-vscode/pull/2839))
+- experimental option to perform remote query validation ([PR #2847](https://github.com/forcedotcom/salesforcedx-vscode/pull/2847))
+- do not emit message from custom editor to webview if soql has not changed ([PR #2818](https://github.com/forcedotcom/salesforcedx-vscode/pull/2818))
```

Comment:
Each entry should start with a capital letter.

Reply:
Acronyms all caps like SOQL, API, URL, etc.

Reply:
Fixed